src/transpilation/transpiler.mjs

```diff
@@ -91,18 +99,22 @@ async function bundle(input, options = {}) {
  * babel plugin configuration for the chosen transpilation mode.
  * @private
  * @param {Array}  plugins - An array of rollup plugins.
- * @param {('NONE'|'DEPENDENCIES'|'ALL')} transpileMode - An enum from `TRANSPILE_MODES` (default `'DEPENDENCIES'`)
+ * @param {('NONE'|'SCOPED'|'ALL')} transpileMode - An enum from `TRANSPILE_MODES` (default `'SCOPED'`)
  * @returns {Array} An array of rollup plugins.
  */
-function configureBabelForTranspileMode(transpileMode, files) {
+async function configureBabelForTranspileMode(transpileMode, transpileIgnores) {
   let babelPluginWithConfig;
   const defaultConfig = {
     babelHelpers: "bundled",
-    exclude: [USER_SCRIPTS_GLOB, POLYFILLS_REGEX],
-    configFile: path.resolve(__dirname, "transpilation/babel.config.json"),
+    extensions: [".js", ".jsx", ".es6", ".es", ".mjs"], // this is same as default
+    // TODO: automatically exclude the `node_modules/core-js` directory from being transpiled
+    // in case the user adds polyfills.  If they use a different lib we can have them
+    // pass it in with the ignore flag
+    exclude: [].concat(transpileIgnores),
+    presets: [babelPreset],
```

Comment:
**Annotation:**
It was a bit awkward to have babel find its config file in the code here, and I learned that you can just pass any config for the babel library (which does transpilation) here programatically.

This opens us up to allow users of the module to pass other babel options to merge with the defaults later which matches with the "do the right thing but provide a powerful escape hatch" philosophy of the library.